multivac/gather_job_data.py

```diff
@@ -166,6 +182,8 @@ def write_csv(self):
             'started_at',
             'completed_at',
             'platform',
+            'commit_sha',
+            'branch',
```

Comment:
Platform and runner label go together for a reason. Please move commit and branch up or down. Branch seems more important, because we can group by branch, so it should go first.

Reply:
done